tldw_Server_API/app/api/v1/endpoints/writing_manuscripts.py

```
@@ -819,3 +879,1366 @@ async def delete_scene(
+
+
+@router.post(
+    "/scenes/{scene_id}/analyze",
+    response_model=list[ManuscriptAnalysisResponse],
+    summary="Run AI analysis on a scene",
+    tags=["manuscripts"],
+)
+async def analyze_scene(
+    scene_id: str,
+    payload: ManuscriptAnalysisRequest,
+    db: CharactersRAGDB = Depends(get_chacha_db_for_user),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)